packages/client/src/types.ts

```diff
@@ -34,6 +34,9 @@ export interface WorkflowExecutionDescription {
   runId: string;
   taskQueue: string;
   status: { code: proto.temporal.api.enums.v1.WorkflowExecutionStatus; name: WorkflowExecutionStatusName };
+  /**
+   * This can safely be converted to a number with `.toNumber()`, as the maximum value is 50,000.
+   */
   historyLength: Long;
```

Comment:
This is stupid, let's just fix this, it's a bug, should've been a number in the first place.